lib/spack/spack/test/cmd/view.py

```diff
@@ -31,6 +31,19 @@
 view = SpackCommand('view')
 
 
+def test_view_symlink_hardlink(
+        tmpdir, builtin_mock, mock_archive, mock_fetch, config,
+        install_mockery):
+    install('libdwarf')
+    for symlink in [True, False]:
```

Comment:
IMO this (and indexing by boolean on the next line) is a bit esoteric: I'd prefer `for symlink, cmd in [(True, 'symlink'), (False, 'hardlink')]`

Reply:
I think this is a good place to use parametrized tests.  Grep around for `pytest.mark.parametrized` in the other tests and you can probably find an example -- you can let `pytest` do this work for you and just refer to parameters.